# Array of skills used for symbolic suggestions
Header header

Skill[] skills